### ISSUE TITLE
[YSQL] yb-admin compact_table not working

### ISSUE THREAD

**Issue by ddorian** [yugabyte/yugabyte-db#3392]
Steps to reproduce:
```
./yb-ctl create
./ysqlsh
```

```
yugabyte=# create database kong2;
yugabyte=# \c kong2
kong2=# create table test(id bigserial primary key);
```
```
./yb-admin compact_table ysql.kong2 test
I0119 09:08:27.858789  6327 yb-admin_client.cc:1266] Resolving namespace id for 'kong2' of type 'ysql'
Error: Not found (yb/master/catalog_manager.cc:2703): Unable to compact table kong2.test: Object does not exist: table_name: "test" namespace { id: "00004100000030008000000000000000" name: "kong2" database_type: YQL_DATABASE_PGSQL }
```

**Comment by d-uspenskiy** [comment]
Problem relates to YSQL tables only. Reason is that `CatalogManager` do not allow to search YSQL tables by name as table name may be non unique in context of single DB (due to different schema's).
Issues will be fixed in context of #1476

**Comment by d-uspenskiy** [comment]
Fixed by https://github.com/yugabyte/yugabyte-db/commit/e6318c0b3711bbacc7c1503029e28fd776c3b30b
